dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/io/readers/SubSnpCoreFieldsRowMapper.java

```diff
@@ -84,6 +84,8 @@
 
     public static final String BATCH_COLUMN = "batch_name";
 
+    public static final String LOAD_ORDER_COLUMN = "load_order";
```

Comment:
We are not mapping this column value against a field in the object, we are just using it in the ORDER BY clause, so, technically, this constant can be in the Reader class.

Reply:
I prefer having all them together, as some columns may get used/unused, and keeping them together will avoid silly movings.